lib/db/mysql/index.js

```diff
@@ -12,6 +12,7 @@ const MysqlPatcher = require('mysql-patcher');
 const config = require('../../config');
 const encrypt = require('../../encrypt');
 const P = require('../../promise');
+const scope = require('../scope');
 const Scope = require('../../scope');
```

Comment:
@divyabiyani hm the `scope` and `Scope` would be really confusing, we need to rename of them